File: SuperiorPackGroup/Reports/TransportationXtraReport.vb
Public Class TransportationXtraReport

    Private Sub expirationDateDetailXrLabel_BeforePrint(sender As Object, e As Printing.PrintEventArgs) Handles expirationDateDetailXrLabel.BeforePrint

        If expirationDateDetailXrLabel.Tag Is Nothing Then
            expirationDateDetailXrLabel.Text = Format(GetCurrentColumnValue("ExpirationDate"), "MM/dd/yyyy")
        Else
            expirationDateDetailXrLabel.Text = Format(GetCurrentColumnValue("ExpirationDate"), expirationDateDetailXrLabel.Tag.ToString)
        End If

    End Sub

End Class

File: SuperiorPackGroup/Reports/TransportationXtraReport.cs
//INSTANT C# NOTE: Formerly VB project-level imports:
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;

namespace SuperiorPackGroup
{
	public partial class TransportationXtraReport
	{

		public TransportationXtraReport()
		{
			InitializeComponent();
		}

		private void expirationDateDetailXrLabel_BeforePrint(object sender, System.Drawing.Printing.PrintEventArgs e)
		{

			if (expirationDateDetailXrLabel.Tag == null)
			{
				expirationDateDetailXrLabel.Text = GetCurrentColumnValue("ExpirationDate").ToString();
			}
			else
			{
				expirationDateDetailXrLabel.Text = Microsoft.VisualBasic.Strings.Format(GetCurrentColumnValue("ExpirationDate"), expirationDateDetailXrLabel.Tag.ToString());
			}

		}

	}
}